tests/test_utils.py

```diff
@@ -125,6 +125,19 @@ def forward(self, prepared_batch):
         return model_output
 
 
+class SimpleModelLogits(torch.nn.Module):
```

Comment:
Inherit the SimpleModel and add stuff on top of it? (MSELoss doesn't even need to be assigned to module, you can directly use functional: https://pytorch.org/docs/stable/nn.functional.html#mse-loss)